### PR TITLE
Fix toolbar dropdown menu

### PR DESCRIPTION
**Pull request by zBritva**

### CODE REVIEW

**Comment by zBritva** [comment]
@danmarshall or @stopyoukid could you review the PR. It's quick fix for serious regression found today.

File: src/app/views/tool_bar.tsx [hunk 1]
```diff
@@ -468,33 +465,40 @@ export class MultiObjectButton extends ContextedComponent<
...
           className="el-dropdown"
-          ref={e => {
+          ref={(e) => {
             if (this.props.compact) {
               return;
             }
             this.refButton = e as any;
           }}
-          onClick={() => {}}
+          onClick={!this.props.compact ? () => openContextMenu() : null}
```
Comment (stopyoukid) [inline src/app/views/tool_bar.tsx:501]: This could reference the `onClick` function you created earlier

Comment (zBritva) [reply src/app/views/tool_bar.tsx:501]: No. logic is opposite

Comment (zBritva) [reply src/app/views/tool_bar.tsx]: I updated functions to clarify logic

**Review by stopyoukid — APPROVED** [review]

**Pull request merged**